internal/io/file/file_source.go

```diff
@@ -276,7 +275,7 @@ func (fs *FileSource) publish(ctx api.StreamContext, file io.Reader, consumer ch
 		ctx.GetLogger().Debug("Sending tuples")
 		for _, m := range resultMap {
 			select {
-			case consumer <- api.NewDefaultSourceTupleWithTime(m, meta, rcvTime):
+			case consumer <- api.NewDefaultSourceTupleWithTime(m, meta, conf.GetNow()):
```

Comment:
Can we let the first recv time begins from the very start?

Reply:
ok, I will fix it